i18n/sync_tx_src.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+const transifex = require('transifex');
+
+// Globals
+const PROJECT = 'scratch-editor';
+const RESOURCE = 'blocks';
```

Comment:
Eventually it would be nice if this lived in scratch-l10n, so maybe for now we should generalize and pass these in as arguments, along with the path to the JSON file, and then call this with those arguments in package.json?